<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
 background-color: rgb(255,255,255)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ScreenshotButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>750</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
</string>
    </property>
    <property name="text">
     <string>Scr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>690</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
	border-radius: 25px;  
}
</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vertexes_text">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>764</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Vertexs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vertexes_num">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>764</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Polygons_num">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>764</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Polygons_text">
    <property name="geometry">
     <rect>
      <x>673</x>
      <y>764</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Polygons:</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileName_1">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>764</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="FileName_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>764</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>261</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
    border: 1px solid rgb(255, 255, 255);
    background: rgb(255, 255, 255);
}

QTabWidget::tab-bar:top {
    top: 1px;
}

QTabBar::tab {
	border: 1px solid rgb(255, 255, 255);
	color: black;
}


QTabBar::tab:selected {
    background:  rgb(255, 255, 255);
}

QTabBar::tab:!selected {
background:  rgb(255, 255, 255);
	 color: rgb(120, 120, 120);
}

QTabBar::tab:top:!selected {
    margin-top: 3px;
}

QTabBar::tab:bottom:!selected {
    margin-bottom: 3px;
}

QTabBar::tab:top{
     font: 700 22pt &quot;Ubuntu&quot;;
    min-width: 8ex;
    margin-right: -1px;
    padding: 1px 1px 1px 1px;
}

QTabBar::tab:bottom {
     font: 700 22pt &quot;Ubuntu&quot;;
    min-width: 8ex;
    margin-left: -100px;
    padding: 1px 1px 1px 1px;
}


QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one {
    margin-right: 0;
}

QTabBar::tab:left:!selected {
    margin-right: 3px;
}

QTabBar::tab:right:!selected {
    margin-left: 3px;
}

QTabBar::tab:left, QTabBar::tab:right {
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;
}

QTabBar::tab:left:selected {
    border-left-color: none;
}

QTabBar::tab:right:selected {
    border-right-color: none;
}

QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one {
    margin-bottom: 0;
}

</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Display</string>
     </attribute>
     <widget class="QLabel" name="Moving">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Moving</string>
      </property>
     </widget>
     <widget class="QLabel" name="mx">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>62</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLabel" name="my">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>99</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
     <widget class="QLabel" name="mz">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>136</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>z</string>
      </property>
     </widget>
     <widget class="QSlider" name="MX_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>73</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="MY_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>111</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="MZ_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>146</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="RZ_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>302</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="RY_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>267</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="ry">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>255</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>y</string>
      </property>
     </widget>
     <widget class="QLabel" name="rz">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>292</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>z</string>
      </property>
     </widget>
     <widget class="QLabel" name="rx">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>218</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLabel" name="Rotate">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>176</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
     <widget class="QSlider" name="RX_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>229</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Scale">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>333</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
     <widget class="QSlider" name="Scale_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>378</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Scale_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>418</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Projection type</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="Scale_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>371</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="MX_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>65</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="MY_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>103</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="MZ_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>140</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="RX_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>221</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="RY_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>259</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="RZ_spinBox">
      <property name="geometry">
       <rect>
        <x>197</x>
        <y>296</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="Parallel_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>470</y>
        <width>117</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Parallel</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="Central_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>513</y>
        <width>115</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Central</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Style</string>
     </attribute>
     <widget class="QLabel" name="Moving_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Colors</string>
      </property>
     </widget>
     <widget class="QLabel" name="Rotate_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>176</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Line style</string>
      </property>
     </widget>
     <widget class="QLabel" name="Rotate_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>335</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Vertex style</string>
      </property>
     </widget>
     <widget class="QLabel" name="Vertex_size">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>518</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Vertex size</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Line_pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 0px solid gray;
	border-radius: 25px;  
}
</string>
      </property>
      <property name="text">
       <string>Line</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Vertex_pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>98</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 0px solid gray;
	border-radius: 25px;  
}
</string>
      </property>
      <property name="text">
       <string>Vertex</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Background_pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>133</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
   border: 0px solid gray;
	border-radius: 25px;  
}
</string>
      </property>
      <property name="text">
       <string>Background</string>
      </property>
     </widget>
     <widget class="QLabel" name="Line_Width_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>308</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Line width</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="line_width">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>305</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_vertex_size">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>520</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="choose_color_line">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>58</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
     <widget class="QPushButton" name="choose_color_vertex">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>93</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
     <widget class="QPushButton" name="choose_color_background">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>128</y>
        <width>70</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Solid_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>224</y>
        <width>86</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Solid</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="Dashed_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>267</y>
        <width>117</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Dashed</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_2</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="Non_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>384</y>
        <width>77</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Non</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_3</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="Circle_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>427</y>
        <width>94</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Circle</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_3</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="Square_radioButton">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>470</y>
        <width>111</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>Square</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_3</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>1101</width>
      <height>750</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Apply_button">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>635</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
	border-radius: 25px;  
}
</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RecordingButton">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>750</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
}
</string>
    </property>
    <property name="text">
     <string>Rec</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
